forum/qa-plugin/q2a-change-username-limit/q2a-changeusernamelimit-widget.php

```diff
@@ -22,10 +22,12 @@ public function output_widget($region, $place, $themeobject, $template, $request
             $history = $this->loadHistoryFromDatabase($user);
 
             if (isset($history)) {
-                $themeobject->output('<h2>' . qa_lang('plugin_username_limit/history_title_label') . '</h2>');
+                $themeobject->output('<div class="qa-part-message-list">');
+                $themeobject->output('<h2><a name="history">' . qa_lang('plugin_username_limit/history_title_label') . '</a></h2>');
                 $themeobject->output('<ul class="q2a-change-username-history-list">');
```

Comment:
Skoro ta [lista jest sortowalna](url), to zamiast `<ul>` bardziej pasuje użyć `<ol>`.

Reply:
ok